packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -425,20 +421,9 @@ ${bodyDetail}`
         result,
         configuration
       );
-      // Check for removing region tags in use.
-      const removeViolations = await checkRemovingUsedTagViolations(
```

Comment:
It makes me slightly nervous that we're able to delete this code with no test changes :)  LGTM assuming CI is passing.  Can we get an issue in the tracker that outlines what's broken, and reminds us to add this back though?

Reply:
Ya, there was no `calledOnce` type assert ;)

Will open a new issue about enabling this.

Reply:
https://github.com/googleapis/repo-automation-bots/issues/1147